### DISCUSSION TITLE
Thumbnails duplication

### DISCUSSION THREAD

**Discussion by StensenE** [igordanchenko/yet-another-react-lightbox#28] [Q&A] [answered]
I have two images, but they duplicates to a number of 5 thumbnails instead. I want to show two thumbnails in this case.
Image:
![image](https://user-images.githubusercontent.com/22217945/194566065-b061f4fa-e608-457a-a68e-83c1b69e7bec.png)

Navigation arrows are not showing up.
What if I had 10 images. How to show the same number in Thumbnails?
I can't find any of this in the documentation.

**Comment by igordanchenko** [comment, marked as answer]
I believe this question was addressed in #21

**Reply by StensenE** [reply]
Yes, but it doesn't work. I tried that but still get 1+ thumbnail.

**Reply by igordanchenko** [reply]
Setting `carousel.finite` to `true` will force 2 thumbnails in this case.

```jsx
<Lightbox
  carousel={{
    preload: 1,
    finite: true
  }}
  ...
/>
```

**Comment by igordanchenko** [comment]
> Navigation arrows are not showing up.

Please share a live demo (preferably, [code sandbox](http://codesandbox.io/)) reproducing this issue.
